control-plane/grpc/src/operations/replica/traits.rs

```diff
@@ -71,7 +72,7 @@ impl From<replica::Replica> for Replica {
             pool: replica.pool_id.into(),
             thin: replica.thin,
             size: replica.size,
-            share: replica::Protocol::from_i32(replica.share).unwrap().into(),
+            share: common::Protocol::from_i32(replica.share).unwrap().into(),
```

Comment:
Maybe we need a tryFrom here to avoid unwrapping

Reply:
I will take this up on the next PR, it would include changes at several places.